### PR TITLE
Additional page template examples

### LINKED ISSUE #27: Multiple pages to demonstrate different layouts and basic navigation
We need more pages in the uikit-starter:
- Add code comments so it's easy to follow and read
- home page
- basic page
- form page

### PR DESCRIPTION
**Pull request by Xochi**
fixes #27

### CODE REVIEW

File: docs/landingpage/subpage.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+		<a class="au-skip-link__link" href="#nav">Skip to main navigation</a>
+	</nav>
+
+	<!-- HEADER -->
+	<header class="au-header au-header--dark" role="banner">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-9">
+					<a class="au-header__brand" href="#">
+						<img class="au-header__brand-image" alt="Insert alternate text here" src="./assets/img/header-logo-agov.png">
```
Comment (sukhrajghuman) [inline docs/landingpage/subpage.html]: Should be `src="../assets/img/header-logo-agov.png"`

File: docs/index.html [hunk 2]
```diff
@@ -105,104 +104,67 @@ <h1 class="au-header__heading">Example Site</h1>
...
-						Examples
-					</button>
-					<button class="au-btn">
-						Get Started
-					</button>
+				<div class="col-sm-12 col-md-6">
+					<h2>Welcome to the design system starter site</h2>
+					<p>If you inspect the html for this page you will see that the <code>&lt;body&gt;</code> tag is using a class called <code>au-grid</code>. This class let's you use the design system grid which is based on 12 columns.</p>
+					<p>Use button classes such as <code>au-btn</code> to style your interactive calls to action.</p>
+					<a class="au-btn au-btn--secondary" href="https://designsystem.gov.au/components/grid-12/">Read about grid</a>
```
Comment (sukhrajghuman) [inline docs/index.html]: I would reword to something like this:

If you inspect the HTML for this page, you will the the CSS class `au-grid` is applied to the `<body>` element. This CSS class lets you use the design system grid, which allows up to 12 columns across a page

thoughts?

File: docs/landingpage.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+				</section>
+				<section class="col-xs-12 col-sm-6 col-md-4 col-lg-4 card card__linked">
+					<a href="/landingpage/subpage.html">
+						<h2 class="au-display-lg">Sub page</h2>
+						<p>This is a card that links to the subpage.</p>
+						<i class="card__carat"></i>
+					</a>
+				</section>
+				<section class="col-xs-12 col-sm-6 col-md-4 col-lg-4 card card__linked">
+					<a href="/landingpage/subpage.html">
```
Comment (sukhrajghuman) [inline docs/landingpage.html]: Maybe make a `subpage2.html`?  It would also make it easier for our users, so they don't have to create another page

File: docs/index.html [hunk 4]
```diff
@@ -105,104 +104,67 @@ <h1 class="au-header__heading">Example Site</h1>
...
+	<!-- MAIN BODY -->
+	<main class="container-fluid au-body main-content">
+		<div class="container-fluid">
+			<div class="row">
+				<!-- The 2 classes below col-sm-6 and col-md-4 set the width of the section based on the viewport width -->
+				<section class="col-sm-6 col-md-4">
+					<div class="content-card">
+						<img class="au-responsive-media-img" src="//via.placeholder.com/350x200" alt="Placeholder image description">
+						<h3>Use the grid system</h3>
+						<p>The <code>col-md-4</code> class (the md stands for medium, the 4 means that this column will take up 4 of 12 grid spaces) on this <code>&lt;section&gt;</code> tag turns it into one of 3 columns on a screen greater than 992px. You will see the other class <code>col-sm-6</code> changes the width of the section based on the size of the screen the website is being viewed on.</p>
```
Comment (sukhrajghuman) [inline docs/index.html]: with the `<p>` inside the cards do you think it would be better to justify the text? Looks cleaner in my opinion. Adding following CSS to the `<p>` 

`text-align: justify`

we get:




instead of

Comment (sukhrajghuman) [reply docs/index.html]: also maybe it's me,  but do the columns feel a little too close together?

File: docs/about.html [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+	<div class="container-fluid au-body main-content">
+		<div class="row">
+			<div class="col-md-12">
+				<nav class="au-breadcrumbs" aria-label="breadcrumb">
+					<ul class="au-link-list au-link-list--inline">
+						<li>
+							<a href="#">Home</a>
+						</li>
+						<li>
+							<a href="#">About</a>
```
Comment (sukhrajghuman) [inline docs/about.html]: This shouldn't be in an anchor tag, since we are already on the about page?

File: docs/newpage.html [hunk 6]
```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
```
Comment (sukhrajghuman) [inline docs/newpage.html]: I thought the breadcrumbs and side nav on the `newpage.html` was confusing.

I guess it's only an example, but `newpage.html` is a parent page. So the breadcrumbs should just be:

Home > New Page.

Same as the side-nav.

The way you did sidenav and breadcrumbs on the landing page and subpages made a lot of sense though :)

**Comment by Xochi** [comment]
Hey @sukhrajghuman the changes have been implemented based on your feedback.

**Comment by sukhrajghuman** [comment]
Hey @Xochi it looks nice man! The only other thing is I'm unsure about the cards in the Landing page, since these aren't one of our components. Thoughts @alex-page

**Comment by alex-page** [comment]
@sukhrajghuman don't forget about clearly showing required fields:
https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html
